CMakeLists.txt

```diff
@@ -473,6 +465,33 @@ else()
     find_package(GDAL)
   endif()
 
+  FIND_PACKAGE(Lua 5.2 EXACT)
```

Comment:
The check here was because of the Lua 5.3 memory issue and not specifically for Luabind - until we confirm the Lua 5.3 memory issues are gone simply by using sol we should keep it.